pkg/registry/apis/provisioning/jobs/persistentstore.go

```diff
@@ -121,7 +114,14 @@ func (s *persistentStore) Claim(ctx context.Context) (job *provisioning.Job, rol
 		if job.Labels == nil {
 			job.Labels = make(map[string]string)
 		}
-		job.Labels[LabelJobClaim] = strconv.FormatInt(s.clock().UnixMilli(), 10)
+		meta, err := utils.MetaAccessor(job)
+		if err != nil {
+			return nil, nil, apifmt.Errorf("failed to get meta accessor for job: %w", err)
+		}
+		meta.SetManagerProperties(utils.ManagerProperties{
```

Comment:
interesting -- i don't think this realy make sense becaue the job is not saved in the repository (and that is what it is checking for here)